UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Controllers/RadioNavigationalWarningsAdminController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using UKHO.MaritimeSafetyInformation.Common.Logging;
+using UKHO.MaritimeSafetyInformation.Common.Models.RadioNavigationalWarning;
+using UKHO.MaritimeSafetyInformation.Web.Services.Interfaces;
+
+namespace UKHO.MaritimeSafetyInformation.Web.Controllers
+{
+    public class RadioNavigationalWarningsAdminController : BaseController<RadioNavigationalWarningsAdminController>
+    {
+        private readonly IRnwService _rnwService;
+        private readonly ILogger<RadioNavigationalWarningsAdminController> _logger;
+
+        public RadioNavigationalWarningsAdminController(IHttpContextAccessor contextAccessor,
+                                                        ILogger<RadioNavigationalWarningsAdminController> logger,
+                                                        IRnwService rnwService) : base(contextAccessor, logger)
+        {
+            _rnwService = rnwService;
+            _logger = logger;
+        }
+
+        public async Task<IActionResult> Index(int pageIndex = 1, int warningType = 0, string year = "", bool reLoadData = true)
```

Comment:
Please can we use a correct (nullable int) data type for year parameter?

Reply:
Changes are done for 

1. (nullable int) data type for year parameter.
2. (nullable int) data type for warningType parameter.